packages/headless/src/features/facets/range-facets/generic/range-facet-reducers.ts

```diff
@@ -11,6 +11,7 @@ export const defaultRangeFacetOptions: RangeFacetOptionalParameters = {
   injectionDepth: 1000,
   numberOfValues: 8,
   sortCriteria: 'ascending',
+  rangeAlgorithm: 'equiprobable',
```

Comment:
The current default is even, switching it to `equiprobable` by default would be a bit breaking, as some customers might be using automatic ranges already. Should we change the default @samisayegh ?

Reply:
I see you answered already 😄  https://github.com/coveo/ui-kit/pull/843#discussion_r641714394